Formula/kubeless.rb

```diff
@@ -3,30 +3,43 @@ class Kubeless < Formula
   homepage "https://github.com/kubeless/kubeless"
   url "https://github.com/kubeless/kubeless/archive/v0.2.3.tar.gz"
   sha256 "e1b1c3b9498ae42b9f2ca7530506fcb37a136bdffb2af7957c3041a93502972a"
-  bottle :unneeded
-
   depends_on "go" => :build
+  depends_on "kubernetes-cli" => :recommended
+
+  def commit
+    if @commit.nil?
```

Comment:
Let's just use the `git` checkout if it absolutely needs a `commit` rather than starting to do things like this, which absolutely nobody new to Ruby is going to understand.

Reply:
okay, I was unaware of that option